tests/e2e/cucumber/features/smoke/link.feature

```diff
@@ -59,8 +59,7 @@ Feature: link
     And "Alice" opens the "files" app
     When "Alice" copies quick link of the resource "folderPublic" from the context menu
     And "Anonymous" opens the public link "Link"
-    And "Anonymous" closes the editor
-    And "Anonymous" downloads the following public link resources using the single share view
+    And "Anonymous" downloads the following public link resources using the sidebar panel
```

Comment:
worked for oCIS but oC10 demands the `closes the editor` step. Was this difference in UI behavior intended? If so then we have to either split the test or implement separate logic for two servers.

Reply:
looks like oCIS behavior is the intended one from https://github.com/owncloud/web/issues/9184